packages/sdk/src/css.ts

```diff
@@ -201,21 +203,33 @@ export const generateCss = ({
       }
       classList.push(className);
     }
-    const rule = sheet.addNestingRule(`.${className}`, descendantSuffix);
+    const rule = userSheet.addNestingRule(`.${className}`, descendantSuffix);
     rule.applyMixins(values);
     instanceByRule.set(rule, instanceId);
   }
 
+  const fontCss = fontSheet.cssText;
+  // render presets inside of cascade layer to let user completely override all properties
+  // user agent (browser) styles work in the same way
+  // for example a { color: black } overrides a:visited as well
+  const presetCss = presetSheet.cssText.replaceAll(
+    "@media all ",
+    "@layer presets "
+  );
```

Comment:
that doesn't look right, should be part of the engine. properly and specified the right way ... e.g. instead of     presetSheet.addMediaRule("presets") presetSheet.addLayer('presets')

Reply:
It's not user facing feature and only done inside of generateCss function. Shouldn't bring any side effects.

Reply:
why doing it at all then? anyways if you insist - fat fat todo comment please with all that stuff you just said

Reply:
Sure. With more usage of this feature we will design the api properly. For now requirements are not clear except for this little use case.

> why doing it at all then?

The feature fits this case well and looks better in devtools.